client/my-sites/plans/jetpack-plans/spp/products-grid-spp/index.tsx

```diff
@@ -0,0 +1,224 @@
+	);
+	const sortedProducts = useMemo(
+		() =>
+			sortBy(
+				getProductsToDisplay( {
+					duration,
+					availableProducts,
+					purchasedProducts,
+					includedInPlanProducts,
+				} ),
+				( item ) => getProductPosition( item.productSlug as JetpackProductSlug )
+			),
+		[ duration, availableProducts, includedInPlanProducts, purchasedProducts ]
+	);
+
+	let popularProducts = [] as SelectorProduct[];
+	let otherProducts = [] as SelectorProduct[];
+
+	const allProducts = sortBy( [ ...sortedPlans, ...sortedProducts ], ( item ) =>
+		getProductPosition( item.productSlug as JetpackPlanSlugs | JetpackProductSlug )
+	);
+	popularProducts = allProducts.slice( 0, 3 );
+	otherProducts = allProducts.slice( 3 );
+
+	const scrollToComparison = () => {
+		if ( bundleComparisonRef.current ) {
+			bundleComparisonRef.current?.scrollIntoView( {
+				behavior: 'smooth',
+			} );
+		}
+	};
+
+	const onResize = useCallback( () => {
+		if ( planGridRef ) {
+			const { current: grid } = planGridRef;
+
+			if ( grid ) {
+				const firstChild = grid.children[ 0 ];
+
+				if ( firstChild instanceof HTMLElement ) {
+					const itemCount = Math.round( grid.offsetWidth / firstChild.offsetWidth );
+
+					setPlanRowWrapping( itemCount < popularProducts.length );
+				}
+			}
+		}
+	}, [ planGridRef, popularProducts ] );
+
+	useEffect( () => {
+		onResize();
+		window.addEventListener( 'resize', onResize );
+
+		return () => window.removeEventListener( 'resize', onResize );
+	}, [ onResize ] );
+
+	const showJetpackFree = isInConnectFlow || ( isInJetpackCloud && ! isUrlSiteConnected );
+
+	return (
+		<Experiment name={ SWITCH_PLAN_SIDES_EXPERIMENT }>
+			<section className="products-grid-spp__section">
+				<h2 className="products-grid-spp__section-title">{ translate( 'Most Popular' ) }</h2>
+				<div className="products-grid-spp__filter-bar">
+					<PlansFilterBarI5
+						showDiscountMessage
+						onDurationChange={ onDurationChange }
+						duration={ duration }
+						withTreatmentVariant={ exPlatVariation === SWITCH_PLAN_SIDES_TREATMENT }
+					/>
+				</div>
+				<ul
+					className={ classNames( 'products-grid-spp__plan-grid', {
+						'is-wrapping': isPlanRowWrapping,
+					} ) }
+					ref={ planGridRef }
+				>
+					{ popularProducts.map( ( product ) => (
+						<li key={ product.iconSlug }>
+							<ProductCardI5
+								item={ product }
+								onClick={ onSelectProduct }
+								siteId={ siteId }
+								currencyCode={ currencyCode }
+								selectedTerm={ duration }
+								isAligned={ ! isPlanRowWrapping }
+								featuredPlans={ [
+									PLAN_JETPACK_SECURITY_DAILY,
+									PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+								] }
+							/>
+						</li>
+					) ) }
+				</ul>
+				<div
+					className={ classNames( 'products-grid-spp__more', {
+						'is-detached': isPlanRowWrapping,
+					} ) }
+				>
+					<MoreInfoBox
+						headline={ translate( 'Need more info?' ) }
+						buttonLabel={ translate( 'Compare all' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Compare All Plans' )` *ES Score: 7*